example/resnet50.yml

```diff
@@ -1,7 +1,8 @@
-weight: /home/lym/.modelci/ResNet50/pytorch-torchscript/1.zip
+weight: ~/.modelci/ResNet50/pytorch-torchscript/image_classification/1.zip
```

Comment:
Have you tried to register with this yaml file? I don't think the user-relative path `~/.modelci/...` will be parsed properly by Python

Reply:
I change home directory to `~` to avoid personal information, we could use `/home/modelci/` instead

Reply:
We could use `PosixPath` instead of `Path` when parsing this location to resolve this problem

Reply:
1. It is good to use `<your home dir>`.

2. `PosixPath` does not understand `~`. You can try this:
```python
from pathlib import PosixPath
p = PosixPath('~')
print(p.absolute())
# PosixPath('/<home dir>/Documents/PyCharmProjects/ml-cloud-autoscaler/~')
```

Reply:
How about use `os.path.expanduser` to wrap the path : https://docs.python.org/3/library/os.path.html#os.path.expanduser

Reply:
Nice! You can mark it as a TODO, and open another PR to solve it. Now it is ok to leave it as `<your home dir>` or `/home/modelci` for demostration.

Reply:
OK!